examples/stable_diffusion_xl/gm/helpers.py

```diff
@@ -401,6 +412,7 @@ def init_sampling(
     num_cols=None,
     sampler="EulerEDMSampler",
     guider="VanillaCFG",
+    guidance_scale=5.0,
```

Comment:
Please also modify in `./demo/streamlit_helpers.py`

Reply:
Not sure what you mean, `stable_diffusion_xl/demo/streamlit_helpers.py` [already has](https://github.com/mindspore-lab/mindone/blob/7e4f9cf9bdfeeac4e0877730d9a62b934adba688/examples/stable_diffusion_xl/demo/streamlit_helpers.py#L144) the guidance scale.